scripts/generate_dump

```diff
@@ -1298,11 +1300,7 @@ save_warmboot_files() {
     else
         mkdir -p $TARDIR
         $CP $V -rf /host/warmboot $TARDIR
-
-        ($TAR $V --warning=no-file-removed  -rhf $TARFILE -C $DUMPDIR --mode=+rw \
-            $BASE/warmboot \
-            || abort "${EXT_TAR_FAILED}" "Tar append operation failed. Aborting for safety.") \
-            && $RM $V -rf $TARDIR
+        chmod ugo+rw -R $DUMPDIR/$BASE/warmboot
```

Comment:
>chmod ugo+rw -R $DUMPDIR/$BASE/warmboot [](http://example.com/codeflow?start=8&length=39)

Could you add some code comment?

Reply:
In line `1302` of the previous code, it will add the `warmboot` folder to archive with `--mode=+rw`, so I did the same but with the different command